### PR TITLE
chore: add stop progressive rollout api

### PR DESCRIPTION
**Pull request by cre8ivejp**
Fixes https://github.com/bucketeer-io/bucketeer/issues/742

#### Things done

- Added the `STOPPED` status
- Added the `StoppedBy` property
- Added Stop Progressive Rollout API
- Updated the SQL queries
- Added missing translations for progressive rollout
- Added e2e tests for Stop API
- Added unit tests

### CODE REVIEW

File: pkg/autoops/domain/progressive_rollout.go [hunk 1]
```diff
@@ -227,3 +229,14 @@ func findTargetSchedule(
...
 }
+
+func (p *ProgressiveRollout) Stop(stoppedBy autoopsproto.ProgressiveRollout_StoppedBy) error {
+	if stoppedBy == autoopsproto.ProgressiveRollout_UNKNOWN {
+		return ErrProgressiveRolloutStoopedByRequired
+	}
+	p.StoppedBy = stoppedBy
+	p.Status = autoopsproto.ProgressiveRollout_STOPPED
+	p.StoppedAt = time.Now().Unix()
+	p.UpdatedAt = time.Now().Unix()
```
Comment (kentakozuka) [inline pkg/autoops/domain/progressive_rollout.go:240]:
```suggestion
	now := time.Now().Unix()
	p.StoppedAt = now
	p.UpdatedAt = now
```

**Review by kentakozuka — CHANGES_REQUESTED** [review]
lgtm 👍 
left a nit

File: pkg/autoops/domain/progressive_rollout.go (hunk 1 above)
Comment (cre8ivejp) [reply pkg/autoops/domain/progressive_rollout.go]: Done.

**Review by kentakozuka — APPROVED** [review]
🚀

**Pull request merged**